master/definitions.xml

```diff
@@ -700,7 +687,6 @@
   <property name='color' href='painting.html#ColorProperty'/>
   <property name='color-interpolation' href='painting.html#ColorInterpolationProperty'/>
   <property name='color-rendering' href='painting.html#ColorRenderingProperty'/>
-  <property name='cursor' href='interact.html#CursorProperty'/>
```

Comment:
There still needs to be a `cursor` property definition for auto-linking, but it should have an updated `href` pointing to CSS-UI.

Reply:
Done